### PR TITLE
mgr: fix 'ceph fs status' output divided by zero error

### PR DESCRIPTION
**Pull request by gmayyyha**
```
Error EINVAL: Traceback (most recent call last):
  File "/home/tiger/ceph/src/pybind/mgr/status/module.py", line 314, in handle_command
    return self.handle_fs_status(cmd)
  File "/home/tiger/ceph/src/pybind/mgr/status/module.py", line 174, in handle_fs_status
    ) + "/s"
  File "/home/tiger/ceph/src/pybind/mgr/status/module.py", line 113, in get_rate
    return (data[-1][1] - data[-2][1]) / float(data[-1][0] - data[-2][0])
ZeroDivisionError: float division by zero
```

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
EAFP. what about just catching `ZeroDivisionError`?

**Comment by gmayyyha** [comment]
> EAFP. what about just catching `ZeroDivisionError`?

@sebastian-philipp mds subtree migration.

**Comment by gmayyyha** [comment]
> > Btw, these are also broken then [...]
> 
> @gmayyyha , @sebastian-philipp - would it make sense to fix these in this PR as well?

@LenzGr @sebastian-philipp define a common function to solve all of these?

**Comment by gmayyyha** [comment]
@sebastian-philipp done all.

**Comment by sebastian-philipp** [comment]
needs rebase.

File: src/pybind/mgr/diskprediction/common/clusterdata.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def differentiate(data1, data2):
     # >>> differentiate(*zip(times, values))
     0.5
     """
+    if float(data2[0] - data1[0]) == 0:
+        return 0
     return (data2[1] - data1[1]) / float(data2[0] - data1[0])
```
Comment (sebastian-philipp) [inline src/pybind/mgr/diskprediction/common/clusterdata.py]: Would be nice if you could merge both copies of `differentiate` into a new common file: `src/pybind/mgr/util.py`. Let's fight code duplication inside mgr modules.

**Comment by sebastian-philipp** [comment]
Relates to #25360

File: src/pybind/mgr/diskprediction/common/clusterdata.py (hunk 1 above)
Comment (gmayyyha) [reply src/pybind/mgr/diskprediction/common/clusterdata.py]: done.

**Comment by sebastian-philipp** [comment]
replaced with #26270

**Pull request closed without merging**